### ISSUE TITLE
code-d deletes DUB cache on Windows

### ISSUE THREAD

**Issue by mdparker** [Pure-D/code-d#182]
When I open a folder containing any D modules with code-d enabled, my dub cache gets deleted (the AppData/dub/packages directory specifically). Every time. Close the folder, open it again (or another with D files) and it's possible to watch the cache disappear in Windows Explorer. 

Not a big deal for dependencies pulled from the repository, but for my local dependencies it's a bit of a pain. I have to remember to backup my local-packages.json when I do a `dub add-local`.

**Comment by WebFreak001** [comment]
does it still happen? dub has been upgraded a few times

**Comment by mdparker** [comment]
So I just enabled it, opened a D proejct, and it updated to the latest. The packages directory is still there, but it's hard to say if the issue has gone away as I get the errors saying the client and server failed to start. Closing VS Code and reopening the project caused the following output:

Installing DCD
Downloading from https://github.com/dlang-community/DCD/releases/download/v0.10.2/dcd-v0.10.2-windows-x86.zip to C:\Users\aldac\AppData\Roaming\code-d\bin
Failed installing: std.net.curl.CurlException@std\net\curl.d(4340): Peer certificate cannot be authenticated with given CA certificates on handle 1DF33D8
----------------
0x0153B83F
0x0153B5F2
0x014F410D

I uninstalled and reinstalled code-d and got the same error.

So as of yet, I can't give a definitive answer.
